package.json

```diff
@@ -1,14 +1,31 @@
 {
-  "name": "@erc6900/reference-implementation",
-  "version": "0.8.0-rc.5",
+  "name": "erc6900-reference-implementation",
+  "description": "Reference implementation for [ERC-6900](https://eips.ethereum.org/EIPS/eip-6900).",
+  "version": "0.8.0-rc.6",
```

Comment:
@huaweigu Mind bumping this to 0.8.1 and updating the package name/relevant bits now that we have the erc6900 org on npm?

Reply:
Done https://github.com/erc6900/reference-implementation/pull/205/commits/2bbfacce16d90b9ffb19f1bd63908f0216ce83be

I also published the package [here](https://www.npmjs.com/package/@erc6900/reference-implementation)